ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -275,8 +276,13 @@ Get Model Inference
             IF    "${project_title}" == "${NONE}"
                 ${project_title}=    Get Model Project    ${model_name}
             END
-            ${token}=    Get Access Token via UI    ${project_title}
-            ${curl_cmd}=     Catenate    ${curl_cmd}    -H "Authorization: Bearer ${token}"
+            IF    $token == ${NONE}
+                ${token}=    Get Access Token via UI    ${project_title}    service_account_name=${service_account_name}
+                ...    single_model=${kserve}    model_name=${model_name}
+                ${curl_cmd}=     Catenate    ${curl_cmd}    -H "Authorization: Bearer ${token}"
+            ELSE
+                ${curl_cmd}=     Catenate    ${curl_cmd}    -H "Authorization: Bearer ${token}"
```

Comment:
you could simplify this and move the line outside IF/ELSE

Reply:
True